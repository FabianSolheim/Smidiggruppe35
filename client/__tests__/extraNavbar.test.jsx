import React from "react";
import ReactDOM from "react-dom";
import { MemoryRouter } from "react-router-dom";
import Header from "../src/components/navbar";

describe("Extra Navbar test", () => {
  it("shows extra navbar", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>,
      domElement
    );
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
