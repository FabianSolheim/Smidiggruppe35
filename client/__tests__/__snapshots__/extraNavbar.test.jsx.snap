// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Extra Navbar test shows extra navbar 1`] = `"<header><header class=\\"MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionSticky makeStyles-header-1 css-7e0s21-MuiPaper-root-MuiAppBar-root\\" style=\\"background-color: rgba(255, 238, 238, 0.2); box-shadow: none; margin-bottom: -150px;\\"><div class=\\"MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular makeStyles-toolbar-3 css-1t29gy6-MuiToolbar-root\\"><a href=\\"/\\"><p class=\\"header-logo\\">Meliora <strong>impact</strong></p></a><nav><ul><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/\\">Home</a><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/about\\">About</a><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/contact\\">Contact</a><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/privacy\\">Privacy</a><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/login\\">Log in</a><a class=\\"MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root makeStyles-menuButton-2 css-1e6y48t-MuiButtonBase-root-MuiButton-root\\" tabindex=\\"0\\" style=\\"font-size: 18px; font-family: Josefin Sans; color: white; text-transform: capitalize;\\" href=\\"/Sign up\\">Sign up</a></ul></nav></div></header></header>"`;
